packages/core/src/connectors/aliyun-sms/index.ts

```diff
@@ -27,6 +29,7 @@ export const metadata: ConnectorMetadata = {
     'zh-CN':
       '短信服务（Short Message Service）是指通过调用短信发送API，将指定短信内容发送给指定手机用户。',
   },
+  readme: readFileSync('./README.md', 'utf8'),
```

Comment:
we can check if this file is exists, prevent crashing.

Reply:
Updated, PTAL.